src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -29,51 +29,75 @@ import {
   styleUrls: ['./avatar.component.scss']
 })
 export class SkyAvatarComponent {
-  public get src(): SkyAvatarSrc {
-    return this._src;
-  }
 
+  /**
+   * Indicates whether users can change the image. To select a different image,
+   * users click the image or drag another image on top of it,
+   * much like the `sky-file-drop` component in the
+   * [file attachment module](https://developer.blackbaud.com/skyux/components/fileattachments).
```

Comment:
Should this link point to https://developer.blackbaud.com/skyux/components/file-attachments/file-attachment? I _think_  the `sky-file-drop` component does not apply to single file attachment, so instead of pointing to the landing page for the file attachment stuff, it seems like we should probably point directly to the topic where `sky-file-drop` comes into play. What do you think?

Reply:
Yes, good catch. Changed.